index.html

```diff
@@ -19,23 +19,170 @@
-            <h3 class="dev-intro">I'am a Software Developer</h3>
-            <p class="background">
-                I can help you build a product , feature or website Look through some of my work and experience!
-                If you like what you see and have a project you need coded, do not hestiate to contact me.
-            </p>
-        </div>
-        <ul class="social-media  container">
-            <li><a href="https://github.com/dunnyk"><img src="image/github.png" alt="github icon"></a></li>
-            <li><a href="https://www.linkedin.com/in/daniel-njoroge-8091b3128/"><img src="image/linkedin.png" alt="linkedin icon"></a></li>
-            <li><a href="https://twitter.com/dankama231"><img src="image/twitter.png" alt="twitter icon"></a></li>
-            <li><a href="https://www.instagram.com/danielkamau231/"><img src="image/finger.png" alt="peace finger sign"></a></li>
-            <li><a href="https://medium.com/@danielkamau231"><img src="image/medium.png" alt="M logo"></a></li>
+    <div class="container">
+        <h1 class="logo-message">I'm  Dan</h1>
+        <h3 class="dev-intro">I'am a Software Developer</h3>
+        <p class="background">
+            I can help you build a product , feature or website Look through some of my work and experience!
+            If you like what you see and have a project you need coded, do not hestiate to contact me.
+        </p>
+    </div>
+    <ul class="social-media  container">
+        <li><a href="https://github.com/dunnyk"><img src="image/github.png" alt="github icon"></a></li>
+        <li><a href="https://www.linkedin.com/in/daniel-njoroge-8091b3128/"><img src="image/linkedin.png" alt="linkedin icon"></a></li>
+        <li><a href="https://twitter.com/dankama231"><img src="image/twitter.png" alt="twitter icon"></a></li>
+        <li><a href="https://www.instagram.com/danielkamau231/"><img src="image/finger.png" alt="peace finger sign"></a></li>
+        <li><a href="https://medium.com/@danielkamau231"><img src="image/medium.png" alt="M logo"></a></li>
+    </ul>
+  </section>
+
+
+  <!-- WORK SECTION -->
+
+  <section class="main-wrap">
+    <article class="work-section">
+      <div class="work-title">
+        <h1>My Recent Works</h1><br>
+        <hr class="line"/>
+      </div>
+        <img src="image/Tonic.png" alt="Tonic Image">
+      <div class="multi-post">
+        <h3>Multi-Post Stories</h3>
+        <p>
+         A daily selection of privately personalized reads; no account or sigh-ups
+         required has been the industry's standard dummy
+         text ever since the 1500s, when unkown printer took a standard dummy text.
+        </p>
+      </div>
+      <div class="work-languages">
+        <ul class="work-list">
+          <li>CSS</li>
+          <li>HTML</li>
+          <li>Bootstrap</li>
+          <li>Ruby</li>
         </ul>
-    </section>
+      </div>
+      <button type="button" class="btn">See Project</button>
```

Comment:
- [ ] Good job adding the button interactions please ensure to you use the right colors stated in the Figma design for when the button is clicked or hovered, please do this for all buttons in your design  
- [ ]  Kindly add the adequate spacing for this button, before the next card there seems to be an overlap👍🏿 

 Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![Screenshot_11](https://user-images.githubusercontent.com/69638013/160236821-13fcae8f-f93d-45c9-aae5-b2a7c28fb336.png) | ![Screenshot_10](https://user-images.githubusercontent.com/69638013/160236813-5165f40b-542b-4198-9762-d16b91ce2f95.png)